Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/PressableButtonPlated.prefab

```diff
@@ -128,7 +132,7 @@ PrefabInstance:
         type: 3}
       propertyPath: m_Materials.Array.data[0]
       value: 
-      objectReference: {fileID: 2100000, guid: e676303390b2ee941b658c1d04ccdc57, type: 2}
```

Comment:
Is this a reference we don't need anymore? I'm not sure exactly what it's referring to.

Reply:
The following command yields no results, so I believe that the reference is no longer valid.
> git grep -i e676303390b2ee941b658c1d04ccdc57